src/ui/edit-content/content-attributes/AttachAttributeField.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  fieldMetaPropTypes,
+} from 'redux-form';
+import { FormattedMessage } from 'react-intl';
+
+import RenderButton from 'ui/common/form/RenderButton';
+import attributeShape from './attributeShape';
+
+const AttachAttributeField = ({
+  label,
+  meta,
+  attribute,
+  ...rest
+}) => {
+  const handleAddClick = () => {
+    // TODO: route to Digital Assets page
```

Comment:
Digital Assets Page is implemented and is in master right now, so you can insert it here.